Builds fail on kinetic-devel

When I run `catkin_make install` from `kinetic-devel` it fails with such message
```
CMake Error at leobot/leobot_launch/cmake_install.cmake:51 (file):
  file INSTALL cannot find
  "/home/user/workspace/leobot/base/src/leobot/leobot_launch/launch/display.launch".
Call Stack (most recent call first):
  cmake_install.cmake:124 (include)
```
[Travis build](https://travis-ci.org/ros-ukraine/leobot/builds/360050368?utm_source=github_status&utm_medium=notification) also fails (tested in PR #64 after merging changes from the main branch).

Feel free to assign to me if this issue is actual.

--- a/leobot_launch/CMakeLists.txt
+++ b/leobot_launch/CMakeLists.txt
@@ -8,7 +8,11 @@ catkin_package(
 )
 
 install(FILES
-  launch/display.launch
+  launch/view_urdf.launch
   launch/simulation.launch
+  launch/navigation.launch
+  launch/rviz.launch
+  launch/teleop.launch
+  launch/web_server.launch
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )

